ui/raidboss/data/05-shb/dungeon/matoyas_relict.js

```diff
@@ -0,0 +1,203 @@
+      condition: function(data, matches) {
+        return data.me === matches.target || data.role === 'healer';
+      },
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'Matoyas Mudman Petrified Peat',
+      netRegex: NetRegexes.startsUsing({ id: '5480', source: 'Mudman', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5480', source: 'Matschmann', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5480', source: 'tadboue', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5480', source: 'マッドマン', capture: false }),
+      response: Responses.moveAway('info'),
+    },
+    {
+      id: 'Matoyas Mudman Peat Pelt',
+      netRegex: NetRegexes.ability({ id: '5482', source: 'Mudman', capture: false }),
+      netRegexDe: NetRegexes.ability({ id: '5482', source: 'Matschmann', capture: false }),
+      netRegexFr: NetRegexes.ability({ id: '5482', source: 'tadboue', capture: false }),
+      netRegexJa: NetRegexes.ability({ id: '5482', source: 'マッドマン', capture: false }),
+      alertText: (data, _, output) => output.pullOrb(),
+      outputStrings: {
+        pullOrb: {
+          en: 'Pull orb to an empty hole',
+          ja: '泥団子を四隅の穴に誘導',
+          cn: '诱导泥球到无敌人的风圈',
+        },
+      },
+    },
+    {
+      id: 'Matoyas Mudman Stone Age',
+      netRegex: NetRegexes.startsUsing({ id: '5491', source: 'Mudman', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5491', source: 'Matschmann', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5491', source: 'tadboue', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5491', source: 'マッドマン', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Matoyas Nixie Crash-smash',
+      netRegex: NetRegexes.startsUsing({ id: '598F', source: 'Nixie', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '598F', source: 'Nixchen', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '598F', source: 'nixe', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '598F', source: 'ノッケン', capture: false }),
+      alertText: (data, _, output) => output.awayFrom(),
+      outputStrings: {
+        awayFrom: {
+          en: 'Away from Tank and his Tether',
+          ja: 'タンクや線に離れ',
+          cn: '远离坦克及其连线',
+        },
+      },
+    },
+    {
+      id: 'Matoyas Nixie Shower Power',
+      netRegex: NetRegexes.startsUsing({ id: '5991', source: 'Nixie', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5991', source: 'Nixchen', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5991', source: 'nixe', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5991', source: 'ノッケン', capture: false }),
+      alertText: (data, _, output) => output.moveTo(),
+      outputStrings: {
+        moveTo: {
+          en: 'Move to ...',
```

Comment:
The other languages here feel more fleshed out, so maybe you were going to come back here, but I feel like `Stand in front of empty pipe` or something like that is probably simple enough?

Reply:
A similar situation in Ghimlyt Dark uses "Avoid Wall Lasers", do you think that could apply here?

Reply:
Oh, I don't know how to describe the mechanic so leave it blank, forgot to point it out!
Thanks.

Reply:
So it can be "Avoid Wall Flush"？